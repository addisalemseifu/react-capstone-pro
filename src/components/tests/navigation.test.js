import React from 'react';
import renderer from 'react-test-renderer';
import { MemoryRouter } from 'react-router-dom';
import Navigation from '../Navigation';
import '@testing-library/jest-dom';

describe('Navigation', () => {
  it('should matche DOM Snapshot', async () => {
    const tree = renderer.create(<MemoryRouter><Navigation /></MemoryRouter>).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
