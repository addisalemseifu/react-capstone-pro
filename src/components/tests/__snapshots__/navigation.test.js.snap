// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navigation should matche DOM Snapshot 1`] = `<div />`;
